### DISCUSSION TITLE
View links from discuss.gogs.io

### DISCUSSION THREAD

**Discussion by unknwon** [gogs/gogs#6863] [Show and tell]
_I am not clever enough to migrate and maintain such a complicated software that was used to run discuss.gogs.io, so I've broke it._

Use [Wayback Machine](https://web.archive.org), e.g. https://web.archive.org/web/20180521083825/https://discuss.gogs.io/t/how-to-backup-restore-and-migrate/991

**Comment by zsyborg** [comment]
i can make it work again

**Comment by ChnTamap** [comment]
hello , I cant find https://discuss.gogs.io/t/how-to-use-predefined-label-templates/599 in the WayBack Machine. Can you tell me how to create predefined label templates? Thanks!

**Reply by unknwon** [reply]
Hey, under your custom `conf/label` directory, putting files with format look like the [default](https://github.com/gogs/gogs/blob/main/conf/label/Default), the file name will become the name of the template appear in the dropdown.

e.g. a file "My creative set" (no file extension)

```
#ee0701 creative bug
#cccccc creative duplicate
#84b6eb creative enhancement
#128a0c creative help wanted
#e6e6e6 creative invalid
#cc317c creative question
#ffffff creative wontfix
```
